Cargo.toml

```diff
@@ -18,6 +18,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 url = "2.1"
 failure = "0.1"
+http = "0.2.1"
```

Comment:
Does the reqwest crate not re-export these?

Reply:
sure does!